### PR TITLE
V2 dev

### PR DESCRIPTION
**Pull request by lmsohamdeshmukh**
Bearer token support added

### CODE REVIEW

**Review by lalit-fgr — APPROVED** [review]

File: logicmonitor_data_sdk/configuration.py [hunk 1]
```diff
@@ -61,9 +61,12 @@ class Configuration(six.with_metaclass(TypeWithDefault, object)):
     >>> conf = logicmonitor_data_sdk.Configuration(company="ACCOUNT_NAME", id='API_ACCESS_ID', key= 'API_ACCESS_KEY')
   """
 
-  def __init__(self, company=None, authentication=None, id=None, key=None):
+  def __init__(self, company=None, authentication=None, id=None, key=None,bearerToken=None):
     """Constructor"""
     # Default Base url
+    self._authentication = {}
+    self.bearerflag = False
```
Comment (mukundneharkar) [inline logicmonitor_data_sdk/configuration.py:68]: why we need two fag??

File: logicmonitor_data_sdk/configuration.py [hunk 2]
```diff
@@ -79,23 +82,37 @@ def __init__(self, company=None, authentication=None, id=None, key=None):
       # type = os.environ.get('LM_ACCESS_TYPE', 'LMv1')
```
Comment (mukundneharkar) [inline logicmonitor_data_sdk/configuration.py:82]: Do we have env for this bearer token?

Comment (lalit-fgr) [reply logicmonitor_data_sdk/configuration.py:82]: @lmsohamdeshmukh Can you please respod?

Comment (lmsohamdeshmukh) [reply logicmonitor_data_sdk/configuration.py:82]: Made changes to incorporate LM_BEARER_TOKEN env variable and removed LMV1authflag

**Pull request merged**